rep-I0005.rst

```diff
@@ -0,0 +1,164 @@
+::
+
+   REP: I0005
```

Comment:
Bit nitpicky perhaps, but REPs can only be assigned a nr once they've been accepted and are made *active*.